lms/djangoapps/support/tests/test_views.py

```diff
@@ -342,8 +343,8 @@ def test_update_enrollment_for_ended_course(self, new_mode):
         self.set_course_end_date_and_expiry()
         self.assert_update_enrollment('username', new_mode)
 
-    def test_update_enrollment_with_credit_mode_throws_error(self):
-        """ Verify that enrollment cannot be changed to credit mode. """
+    def test_update_enrollment_with_credit_mode(self):
```

Comment:
I believe we can remove this test now because we are already testing credit mode in `test_update_enrollment_for_all_modes`.